File: samples/java/SupportChat/Server/Configuration/src/main/java/systems/zlink/samples/supportchat/server/configuration/SampleTimings.java
package systems.zlink.samples.supportchat.server.configuration;

import java.time.Duration;

public final class SampleTimings {
    public static final Duration ConnectTimeout = Duration.ofSeconds(5);
    public static final Duration RequestTimeout = Duration.ofSeconds(5);
    public static final Duration IdleTimeout = Duration.ofSeconds(3);
    public static final Duration CloseGraceTimeout = Duration.ofSeconds(2);

    private SampleTimings() {}
}

File: samples/kotlin/SupportChat/Server/Configuration/src/main/kotlin/systems/zlink/samples/kotlin/supportchat/server/configuration/SampleTimings.kt
package systems.zlink.samples.kotlin.supportchat.server.configuration

import java.time.Duration

object SampleTimings {
    val ConnectTimeout: Duration = Duration.ofSeconds(5)
    val RequestTimeout: Duration = Duration.ofSeconds(5)
    val IdleTimeout: Duration = Duration.ofSeconds(3)
    val CloseGraceTimeout: Duration = Duration.ofSeconds(2)
}
